plugins/module_utils/job.py

```diff
@@ -34,12 +34,9 @@ def job_output(job_id=None, owner=None, job_name=None, dd_name=None):
         job_name {str} -- The job name search for (default: {None})
         dd_name {str} -- The data definition to retrieve (default: {None})
 
-    Raises:
-        RuntimeError: When job output cannot be retrieved successfully but job exists.
-        RuntimeError: When no job output is found
-
     Returns:
         list[dict] -- The output information for a list of jobs matching specified criteria.
+        If no job status is found, this will return an empty job code with msg=Job not found
```

Comment:
I know I am being overly critical here but "Job not found" is not equal to "Job Not Found", one has uppercase Not and Found.

Reply:
I get it ... consistency.